### PR TITLE
[mnist]:Delete the unused domain names

### PR DESCRIPTION
**Pull request by DjangoPeng**
Delete the unused domain names of `ps` and `worker`.

### CODE REVIEW

File: tensorflow/tools/dist_test/python/mnist_replica.py [hunk 1]
```diff
@@ -84,9 +84,6 @@
 
 IMAGE_PIXELS = 28
 
-PARAM_SERVER_PREFIX = "tf-ps"  # Prefix of the parameter servers' domain names
-WORKER_PREFIX = "tf-worker"  # Prefix of the workers' domain names
-
```
Comment (DjangoPeng) [inline tensorflow/tools/dist_test/python/mnist_replica.py]: Delete the domain names because the latest `mnist_replica` never use them.

**Comment by caisq** [comment]
@tensorflow-jenkins test this please.

**Comment by caisq** [comment]
LGTM.

**Comment by caisq** [comment]
Merged. Thanks.

**Pull request merged**